CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+Mediapublic is an Open Source project, and feedback, code, documentation, and
+other contributions are all welcome. These are guidelines for new (and
+existing) contributors who want to help.
+
+## Contributing code
+
+We work on the "develop" branch, and update master to the latest stable
+version every so often. When you make changes and file pull requests, please
+submit them against "develop."
+
+1. You'll need to [fork this repository][fork] to your own GitHub account
+1. Once you've forked the repo, clone it
+   `git clone git@github.com:YOURNAME/mediapublic.git`
+1. Make your change and commit it
+1. File a [pull request][pull] and wait!
+
+After filing a pull request, it may take some time for us to review it. If
+there are problems, or something isn't clear we may ask some questions.
+
+#### How to get a pull request accepted
+
+In short:
+
+1. Don't break existing tests or functionality
+1. Write tests for the new code (see Testing)
+1. Write a [good commit message][commit]
+1. Tell us why you made this change
+
+### Testing
+
+We have automated tests that are run on every pull request, and as a rule we
+don't merge code that breaks the tests. For the Python backend, we use
+[tox][tox] which enforces code style, runs our API tests, and tests on multiple
+Python versions.
+
+You can run all the tests before you push code by typing `tox` in the `server`
+directory and it will run all the tests.
+
+We test the Python server using [gabbi][gabbi], which takes YAML definitions of
+requests and runs them to confirm they succeed. For mediapublic, this is
+important because the HTTP API is how the web application communicates with the
+backend server, so it *has* to work all the time. Look [here][gabbitests] to
+see examples of those tests.
+
+## Development Schedule
+
+The development team works in two-week sprints, and track what we're working on
+with [GitHub milestones][milestones]. As issues come in, we tag them as "next
+sprint" if we plan to do them soon (but not immediately). Each sprint has a due
+date associated, and what doesn't get done will be moved to the following
+sprint.
+
+At the beginning of each sprint, we tag issues we plan to work on and
+self-assign tasks. If you'd like to join us, let us know!
+
+## Issue Tags
+
+We have a few tags to help us navigate the issues page:
+
+- "bug" - Bugs that need squashing because something isn't working right.
+- "coding" - Tasks that involve programming, either frontend or backend.
+- "design" - This means feedback is desired. When an issue has this tag, it's
+    often about a future feature or a change to the project.
+- "documentation" - All things documentation. Developer docs, user docs,
+    installation docs, you name it.
+- "duplicate" - Exactly what it says on the tin.
+- "enhancement" - New features, or improvements to existing ones.
```

Comment:
We originally had both "feature" and "enhancement" and at the meeting I think we decided we could use one tag for both.